api-report/merge-tree.api.md

```diff
@@ -1592,6 +1610,9 @@ export const UnassignedSequenceNumber = -1;
 // @public
 export const UniversalSequenceNumber = 0;
 
+// @internal (undocumented)
+export function _validateReferenceType(refType: ReferenceType): void;
```

Comment:
should be able to hide this by adjusting the export statement on merge-tree's index to not export *. That's preferable IMO to exposing it and marking internal.